### PR TITLE
Update build documentation for stratum-deps

### PR DESCRIPTION
**Pull request by ffoulkes**
- Update stratum dependencies build guides to reference the `stratum-deps` repository.

### CODE REVIEW

File: docs/guides/deps/building-stratum-deps.md [hunk 1]
```diff
@@ -9,49 +9,44 @@ This document explains how to build and install the Stratum dependencies.
 > **Note**: For the Intel&reg; IPU E2100, see
 [Building Stratum Dependencies for the ACC](building-acc-stratum-deps.md).
 
-## 1 Prerequisites
+## Prerequisites
 
-### 1.1 CMake
+There a couple of things to do before you build the dependencies.
```
Comment (5abeel) [inline docs/guides/deps/building-stratum-deps.md:14]:
```suggestion
There are a couple of things to do before you build the dependencies.
```

File: docs/guides/deps/building-acc-stratum-deps.md [hunk 2]
```diff
@@ -24,38 +24,28 @@ The Host and Target libraries must be the same version.
 
 ## 2. Preparing the System
 
-There are several things to do before you can build the dependencies.
+There a couple of things to do before you build the dependencies.
```
Comment (5abeel) [inline docs/guides/deps/building-acc-stratum-deps.md:27]:
```suggestion
There are a couple of things to do before you build the dependencies.
```

File: docs/guides/deps/building-acc-stratum-deps.md [hunk 3]
```diff
@@ -136,7 +122,8 @@ dependencies (`--host`) as well as the install prefix (`--prefix`).
 
 ### Target build
 
-Source the file that the defines the [target build environment variables](/guides/es2k/defining-acc-environment.md).
+Source the file that the defines the
```
Comment (5abeel) [inline docs/guides/deps/building-acc-stratum-deps.md:125]:
```suggestion
Source the file that defines the
```

File: docs/guides/deps/building-stratum-deps.md [hunk 4]
```diff
@@ -9,49 +9,44 @@ This document explains how to build and install the Stratum dependencies.
...
+  P4 Control Plane uses OpenSSL instead of BoringSSL.
 
-## 2 Source Code
+## Source Code
 
 There are two ways to obtain the source code: from the open-source repository
 on GitHub, or as a source package.
 
-### 2.1 GitHub Repository
+### Github repository
```
Comment (5abeel) [inline docs/guides/deps/building-stratum-deps.md:30]: `GitHub` with uppercase `H`?

**Review by 5abeel — CHANGES_REQUESTED** [review]

**Comment by ffoulkes** [comment]
Correct on all count. Thanks.

**Review by 5abeel — APPROVED** [review]
LGTM

**Review by bharticemk — APPROVED** [review]
LGTM

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Review by n-sandeep — APPROVED** [review]
LGTM

**Pull request merged**